crates/bevy_core_pipeline/src/core_3d/camera_3d.rs

```diff
@@ -5,21 +5,47 @@ use bevy_render::{
     camera::{Camera, CameraRenderGraph, Projection},
     extract_component::ExtractComponent,
     primitives::Frustum,
-    render_resource::LoadOp,
+    render_resource::{LoadOp, TextureUsages},
     view::VisibleEntities,
 };
 use bevy_transform::prelude::{GlobalTransform, Transform};
 use serde::{Deserialize, Serialize};
 
 /// Configuration for the "main 3d render graph".
-#[derive(Component, Reflect, Clone, Default, ExtractComponent)]
+#[derive(Component, Reflect, Clone, ExtractComponent)]
 #[extract_component_filter(With<Camera>)]
 #[reflect(Component)]
 pub struct Camera3d {
     /// The clear color operation to perform for the main 3d pass.
     pub clear_color: ClearColorConfig,
     /// The depth clear operation to perform for the main 3d pass.
     pub depth_load_op: Camera3dDepthLoadOp,
+    /// The texture usages for the depth texture created for the main 3d pass.
+    pub depth_texture_usages: Camera3dDepthTextureUsage,
```

Comment:
what happens if I set a value without `TextureUsages::RENDER_ATTACHMENT` here? Should we consider it's present even if the user didn't set it?